Samples/SwiftJavaExtractFFMSampleApp/Sources/MySwiftLibrary/MySwiftStruct.swift

```diff
@@ -61,4 +61,9 @@ public struct MySwiftStruct {
   public func makeRandomIntMethod() -> Int {
     return Int.random(in: 1..<256)
   }
+
+  public subscript() -> Int {
+    get { return len }
+    set { len = newValue }
+  }
```

Comment:
What about a case with parameters?

Reply:
I didn't notice I lost them in git stash :/ Fixed